### ISSUE TITLE
Build fails with 'Cannot read property 'path' of undefined' after update from 1.2.1 to 1.2.2

### ISSUE THREAD

**Issue by zhaparoff** [wessberg/rollup-plugin-ts#65]
> xrm-alerts@1.0.11 build:core /home/vsts/work/1/s
> rollup -c

```
/home/vsts/work/1/s/src/ts/alert.ts → dist/alert.js, dist/alert.cjs.js, dist/alert.esm.js...
[!] (plugin Typescript) TypeError: Cannot read property 'path' of undefined
src/ts/alert.ts
TypeError: Cannot read property 'path' of undefined
    at addReferencedFilesToTypeDirective (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:64501:46)
    at /home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:64336:21
    at Map.forEach (<anonymous>)
    at createResolver (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:64329:49)
    at Object.createTypeChecker (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:33256:28)
    at Object.getTypeChecker (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:95754:79)
    at getReferencedFiles (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:97484:43)
    at Object.create (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:97531:41)
    at createBuilderProgramState (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:97869:37)
    at Object.createBuilderProgram (/home/vsts/work/1/s/node_modules/typescript/lib/typescript.js:98464:21)
```

**Comment by wessberg** [comment]
Hey there, I'm sorry to hear that! Can you provide some more details as to how I might be able to reproduce the error?

**Comment by zhaparoff** [comment]
I'm building this repo: https://github.com/zhaparoff/xrm-alerts/tree/dependabot/npm_and_yarn/wessberg/rollup-plugin-ts-1.2.2

Azure build failed, as well as local build on windows machine.

Node: 13.3.0
Npm: 6.13.4

**Comment by wessberg** [comment]
Thanks! I'll take a look.

**Comment by wessberg** [comment]
The problem has been resolved in [v1.2.3](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.3)
